src/controllers/v1.controller.ts

```diff
@@ -304,8 +304,7 @@ export class V1Controller {
     @param.filter(Applications, { exclude: 'where' })
     filter?: FilterExcludingWhere<Applications>
   ): Promise<string | ErrorObject> {
-    const parsedRawData = parseRawData(rawData)
```

Comment:
Having this outside of the try-catch, made our code trigger Loopback http errors thats why sometimes you'd get non JSON RPC errors (not desired).